packages/flutter/lib/src/material/popup_menu_theme.dart

```diff
@@ -0,0 +1,156 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
```

Comment:
I think the analyzer might be complaining about this line being a recursive import. Just import the particular files you need and not the entire material library

Reply:
Done